src/toxcore/messenger/packet/file_control.rs

```diff
@@ -0,0 +1,148 @@
+/*! FileControl struct.
+It is used to control transferring file to a friend.
+*/
+
+use nom::{le_u8, be_u64};
+
+use crate::toxcore::binary_io::*;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
+/// Whether this node is sender or receiver of file data packet
+pub enum SendReceive {
```

Comment:
How about renaming SendReceive => ControlDirection?
@kurnevsky ?

Reply:
Yeah, I did the similar thing with even more complex state: https://github.com/tox-rs/tox/blob/5c57ddd68621b2c87a24e181a0134dce0194966f/src/toxcore/net_crypto/crypto_connection.rs#L144

Reply:
Or maybe TransferDirection?